SimTracker/TrackerMaterialAnalysis/data/dd4hep_trackingMaterialGroups_ForPhaseII.xml

```diff
@@ -65,71 +65,71 @@
       <Parameter name="TrackingMaterialGroup" value="TrackerRecMaterialPhase2PixelBarrelLayer4" />
     </SpecPar>
 
-    <SpecPar name="TrackerRecMaterialPhase2PixelForwardDisk1">
+    <SpecPar name="TrackingMaterialGroupPhase2PixelForwardDisk1">
       <PartSelector path="//pixfwd:Phase2PixelEndcap/pixel:Disc1/pixel:Ring1Disc1/pixel:EModule1Disc1/pixel:EModule1Disc1InnerPixelwafer/pixel:EModule1Disc1InnerPixelActive"/>
       <PartSelector path="//pixfwd:Phase2PixelEndcap/pixel:Disc1/pixel:Ring2Disc1/pixel:EModule2Disc1/pixel:EModule2Disc1InnerPixelwafer/pixel:EModule2Disc1InnerPixelActive"/>
       <PartSelector path="//pixfwd:Phase2PixelEndcap/pixel:Disc1/pixel:Ring3Disc1/pixel:EModule3Disc1/pixel:EModule3Disc1InnerPixelwafer/pixel:EModule3Disc1InnerPixelActive"/>
       <PartSelector path="//pixfwd:Phase2PixelEndcap/pixel:Disc1/pixel:Ring4Disc1/pixel:EModule4Disc1/pixel:EModule4Disc1InnerPixelwafer/pixel:EModule4Disc1InnerPixelActive"/>
       <Parameter name="TrackingMaterialGroup" value="TrackerRecMaterialPhase2PixelForwardDisk1" />
     </SpecPar>
 
-    <SpecPar name="TrackerRecMaterialPhase2PixelForwardDisk2">
+    <SpecPar name="TrackingMaterialGroupPhase2PixelForwardDisk2">
       <PartSelector path="//pixfwd:Phase2PixelEndcap/pixel:Disc2/pixel:Ring1Disc2/pixel:EModule1Disc2/pixel:EModule1Disc2InnerPixelwafer/pixel:EModule1Disc2InnerPixelActive"/>
       <PartSelector path="//pixfwd:Phase2PixelEndcap/pixel:Disc2/pixel:Ring2Disc2/pixel:EModule2Disc2/pixel:EModule2Disc2InnerPixelwafer/pixel:EModule2Disc2InnerPixelActive"/>
       <PartSelector path="//pixfwd:Phase2PixelEndcap/pixel:Disc2/pixel:Ring3Disc2/pixel:EModule3Disc2/pixel:EModule3Disc2InnerPixelwafer/pixel:EModule3Disc2InnerPixelActive"/>
       <PartSelector path="//pixfwd:Phase2PixelEndcap/pixel:Disc2/pixel:Ring4Disc2/pixel:EModule4Disc2/pixel:EModule4Disc2InnerPixelwafer/pixel:EModule4Disc2InnerPixelActive"/>
       <Parameter name="TrackingMaterialGroup" value="TrackerRecMaterialPhase2PixelForwardDisk2" />
     </SpecPar>
 
-    <SpecPar name="TrackerRecMaterialPhase2PixelForwardDisk3">
+    <SpecPar name="TrackingMaterialGroupPhase2PixelForwardDisk3">
       <PartSelector path="//pixfwd:Phase2PixelEndcap/pixel:Disc3/pixel:Ring1Disc3/pixel:EModule1Disc3/pixel:EModule1Disc3InnerPixelwafer/pixel:EModule1Disc3InnerPixelActive"/>
```

Comment:
@ghugo83 - it seems to me that the names could be simplified as well. 

For example:
```
//pixfwd:Phase2PixelEndcap/pixel:Disc2/pixel:Ring1Disc2/pixel:EModule1Disc2/pixel:EModule1Disc2InnerPixelwafer/pixel:EModule1Disc2InnerPixelActive
```
could be simplified as:
```
//PixelEndcapPhase2/PixelDisk2/PixelRing1/PixelEModule1/PixelEInnerWafer/PixelEInnerActive
```
note, it's also should be `Disk` not `Disc`, I think :-)

Reply:
Yes exactly, this is what I have done in the subsequent PR (gonna push it now for Phase 1)

Reply:
Ha was not the namespace but the head volumes that I had done. Will also add the namespaces in the cases it leads to no regression

Reply:
But the biggest effect from several orders of magnitude ended up to remove the regex from trackerPhase1RecoMaterial.xml (the Phase 2 RecoMaterial file has no regex)

Reply:
@ianna Phase 1: when I try to remove a namespace I get issues with old DD:
`Qualify the name with a regexp for the namespace, i.e ".*:name-regexp" !`
Will see whether it is different for Phase 2.

I have just removed the parent volumes in Phase 1 trackerRecoMaterial.xml to see, but it had no visible impact (regex removal instead, made the full DD4hep-based step 1 till event generator time from ~140 s to ~85s, while after removing the parent volumes, I still get ~85s).
I can push the cut of the parents volumes to the other PR though, does not harm.